src/containers/BooksForm.js

```diff
@@ -31,15 +32,19 @@ function BooksForm() {
   };
 
   return (
-    <form onSubmit={onFormSubmit}>
-      <input type="text" id="title" placeholder="enter title" onChange={handleInputChange} value={title} />
-      <select onChange={handleSelectChange} value={category}>
-        {categories.map((cat) => <option key={Math.random() * 1000}>{cat}</option>)}
-
-      </select>
-      <button type="submit">Submit</button>
-      <p>{error}</p>
-    </form>
+    <div className="form-div">
+      <hr />
+      <h3 className="form-title">ADD NEW BOOK</h3>
+      <form onSubmit={onFormSubmit} className="form">
+        <input className="title-input form-control" type="text" id="title" placeholder="Book title" onChange={handleInputChange} value={title} />
+        <select className="form-select" onChange={handleSelectChange} value={category} placeholder="Category">
+          {categories.map((cat) => <option key={Math.random() * 1000}>{cat}</option>)}
+
+        </select>
+        <button className="submit btn btn-primary chapter-btn" type="submit">ADD BOOK</button>
+        <p>{error}</p>
+      </form>
+    </div>
   );
```

Comment:
- [ ] You are almost there with this component, thoroughly inspect the design for color specifications and fonts used so that they are a match.